### ISSUE TITLE
Pairing-plus out-dated

### ISSUE THREAD

**Issue by andrewwhitehead** [hyperledger/aries-bbs-rs#5]
[pairing-plus](https://crates.io/crates/pairing-plus) is the library that implements the bls12-381 curve as well as the hash-to-curve operations. It hasn't been updated in 3 years at this point, and while the bls12_381 library would be a good replacement otherwise, unfortunately pairing-plus [does not match up with the current hash-to-curve spec](https://github.com/algorand/pairing-plus/issues/19) - it implements draft 7. Updating would be a breaking change, but that also restricts the versions of blake2 and hkdf that can be used here. The only solution to allow updating the other dependencies without breaking compatibility of the signatures might be to vendor the pairing-plus dependency.

**Comment by swcurran** [comment]
Can we use the existing library for now, and then consider update options once we have a stable deployment?

**Comment by andrewwhitehead** [comment]
Yes, it still builds and there aren't any known security issues.
